src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age ? (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    ) : null}
```

Comment:
```suggestion
    {person.age && (
      <p className="Person__age">
        {`I am ${person.age}`}
      </p>
    )}
```
If there is nothing to render in the second case - use conditional rendering instead of ternary operator